### PR TITLE
Add two subjects from FreeCol and JGraphT to the precision/recall test suite

### PR DESCRIPTION
**Pull request by ariannab**
A pair of experiments done: net.sf.freecol.common.model.Unit from FreeCol and org.jgrapht.experimental.dag.DirectedAcyclicGraph from JGraphT

### CODE REVIEW

**Comment by mernst** [comment]
Could you please:
 * give the pull request a descriptive name (not just "Dev")?
 * make the tests pass?

Thanks!

**Comment by ariannab** [comment]
I changed the name, but the tests pass on my machine, so I don't know how to fix this problem right now. I'm going to investigate.

**Comment by mernst** [comment]
The problem is formatting.  If you reformat your newly-added files, the tests should pass on Travis.

**Comment by ariannab** [comment]
Thank you so much for the suggestion and sorry for the inconvenience. Fixed it!

File: src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Unit_goal.json [hunk 1]
```diff
@@ -0,0 +1,14431 @@
...
+      "qualifiedName": "boolean",
+      "name": "boolean",
+      "isArray": false
+    },
+    "parameters": [],
+    "paramTags": [],
+    "returnTag": {
+      "comment": "True if this unit can carry goods or other units.",
+      "kind": "RETURN",
+      "condition": "target.unitType.canCarryGoods() || target.unitType.canCarryUnits()"
```
Comment (albertogoffi) [inline src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Unit_goal.json]: I think the comment should be translated as
`target.unitType.canCarryGoods() || target.unitType.canCarryUnits() ? result==true`

Comment (ariannab) [inline src/test/resources/goal-output/freecol-0.11.6/net.sf.freecol.common.model.Unit_goal.json]: You're right. Fixed

**Pull request merged**